services/mgmt/confidentialledger/Cargo.toml

```diff
@@ -29,10 +29,11 @@ env_logger = "0.9"
 all-features = true
 
 [features]
-default = ["package-2022-05-13", "enable_reqwest"]
+default = ["package-2022-09-08-preview", "enable_reqwest"]
```

Comment:
This should remain on `package-2022-05-13` for the default. I'll fix with #1112.